app/src/main/java/com/amaze/filemanager/adapters/glide/RecyclerPreloadModelProvider.java

```diff
@@ -43,17 +43,14 @@ public RecyclerPreloadModelProvider(@NonNull Fragment fragment, @NonNull List<Ic
     @Override
     @Nullable
     public RequestBuilder<Drawable> getPreloadRequestBuilder(IconDataParcelable iconData) {
-        if(!showThumbs) {
```

Comment:
There is a setting to only show resource icons, not thumbs, shouldn't the code for that be present here?

Reply:
The code for that is present in RecyclerAdapter. It's not necessary here.